README.md

```diff
@@ -45,7 +45,8 @@ The `gitlab.rb.j2` template packaged with this role is meant to be very generic
   - Create a `templates\mygitlab.rb.j2` file (just choose a different name from the default template).
   - Set the variable like: `gitlab_config_template: mygitlab.rb.j2` (with the name of your custom template).
 
-    # SSL Configuration.
+# SSL Configuration.
```

Comment:
This should actually be `###` (three hashes), to make it an `h3` level item (since it's a subsection of the variable heading).

Reply:
Fixed